pyproject.toml

```diff
@@ -15,8 +15,9 @@ exclude = ["src/book/_build"]
 [tool.poetry.dependencies]
 python = ">=3.9,<=3.12"
 pyyaml = "^6.0.1"
-gitpython = "^3.1.43"
 pygit2 = "^1.15.1"
+fire = "^0.6.0"
+gitpython = "^3.1.43"
```

Comment:
It looks like `gitpython` was added or moved here. Double checking - was this intentional, would we still need it?

Reply:
I realized for our test repos that they still use `gitpython`. From looking around `pygit` docs, this seems like it may be a pretty big change, but I'll refrence this in a issue for a future PR!